### ISSUE TITLE
Amazon SES Python API

### ISSUE THREAD

**Issue by fabiop** [emencia/emencia-django-newsletter#22]
Hello, first compliments for your projects : )

I'm writing to suggest a possible feature request, I think it would be quite interesting adding Amazon SES (Simple Email Service) capabilities integrating this python API:
https://github.com/pankratiev/python-amazon-ses-api

I would try, to a certain degree, to do it by myself but I'm quite new with Django and I think I 
would need some guidance...

Cheers
Fabio

**Comment by Fantomas42** [comment]
Hi,
thank you for your support.

It's a good idea to integrate it, I will take a look on it, I see what can be done.

Regards,

**Comment by Fantomas42** [comment]
Ok I have looked into the module, it's very interessting.

My idea for implementing Amazon SES, is to write a new django command management in the package, using a subclass of the Mailer object.

What do you think about this idea ?

**Comment by fabiop** [comment]
Thanks for the fast feedback, I think it's a great idea connecting it to the Mailer object, good also to be used in perspective for general purpose inside Django ; )

I'm going to take a look at Django Mailer as well.

Regards

**Comment by xaralis** [comment]
Simple solution till real one is made is to use SMTP relay - our simple implementation can be found in  https://github.com/fragaria/fragapy/tree/master/fragapy/amazon.

**Comment by xaralis** [comment]
Docs live in http://readthedocs.org/docs/fragapy/en/latest/amazon.html.

**Comment by Fantomas42** [comment]
It's a good idea.

Perl rox ! :)

**Comment by xaralis** [comment]
Yes, we are already using it in production with your newsletter app gracefully ;)

**Comment by YuChem** [comment]
Amazon has SMTP interface to their SES service. I have managed making the newsletter to work with it.
Will there be interest in this fixes to be merged to the main branch?
